lambdas/service/lambda-policy.json.template.py

```diff
@@ -146,6 +146,15 @@
                 f"arn:aws:apigateway:{aws.region_name}::"
                 f"/restapis/{aws.api_gateway_id(config.service_name)}/stages/{config.deployment_stage}/exports/oas30"
             ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "dynamodb:Query"
+            ],
+            "Resource": [
+                f"arn:aws:dynamodb:{aws.region_name}:{aws.account}:table/{config.dynamo_failure_message_table_name}",
+            ]
```

Comment:
Instead of adding a new Statement dict here, could the Resource be added to the existing `"Effect": "Allow"` Statement on line 108?

Reply:
I just tested your change and it doesn't mess up the deployment. Plus its cuts a few lines in the commit.

Reply:
Yes, my mistake. I'll remove it.